sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -152,7 +151,7 @@
 	public static final String DATABASE_NAME = "sormas.db";
 	// any time you make changes to your database objects, you may have to increase the database version
 
-	public static final int DATABASE_VERSION = 247;
+	public static final int DATABASE_VERSION = 248;
```

Comment:
There was a mistake by one of our developers in version 247; please increase this to 249 to make it work again